Event.js

```diff
@@ -0,0 +1,186 @@
+import * as L from "./Logger.js";
+import * as Validate from "./Validators.js";
+
+class Event {
+    #state;
+
+    constructor(state) {
+        this.#state = state;
+    }
+
+    /**
+     * Used to send various types of event;
+     *
+     * @param {CountlyEventOptions} options event
+     * @return {string | void} error message or void
+     */
+    sendEvent(options) {
```

Comment:
yea lets go with recordEvent. we will deprecate the old one anyway

Reply:
done